packages/seal/src/Search/Condition/AbstractGroupCondition.php

```diff
@@ -19,12 +19,12 @@
 abstract class AbstractGroupCondition
 {
     /**
-     * @var array<EqualCondition|GreaterThanCondition|GreaterThanEqualCondition|IdentifierCondition|LessThanCondition|LessThanEqualCondition|NotEqualCondition|AndCondition|OrCondition>
+     * @var array<EqualCondition|GeoBoundingBoxCondition|GeoDistanceCondition|GreaterThanCondition|GreaterThanEqualCondition|IdentifierCondition|InCondition|LessThanCondition|LessThanEqualCondition|NotEqualCondition|NotInCondition|AndCondition|OrCondition>
```

Comment:
If I remember correctly GeoBoundingBoxCondition|GeoDistanceCondition are not supported inside And and Or conditions because some search engines have special root level filters for it only and you can not combine it with OR. So I did keep that out of AbstractGroupCondition.

```suggestion
     * @var array<EqualCondition|GreaterThanCondition|GreaterThanEqualCondition|IdentifierCondition|InCondition|LessThanCondition|LessThanEqualCondition|NotEqualCondition|NotInCondition|AndCondition|OrCondition>
```